client/src/index.jsx

```diff
@@ -86,11 +93,75 @@ class Zillow extends React.Component {
         main: [],
         group: []
       },
+      value: '',
       mainPage: true,
       index: {}
     };
+    this.deletePhoto = this.deletePhoto.bind(this);
+    this.closePhotoModal = this.closePhotoModal.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  updatePhoto(e) {
+    var path = window.location.pathname.split('/');
+    var house = path[1];
+    var self = this;
+    var url = this.state.value;
+    $.ajax({
+      method: 'PUT',
+      url: `http://localhost:3002/gallery/${house}/photo/${this.state.photoSelected}/url/${this.state.value}`,
```

Comment:
You can do a similar destructuring operation to this.state to help shorten the url string. I would also just assign the url string to a variable. Here's an example:

`const { photoSelected, value } = this.state`
`var url =`http://localhost:3002/gallery/${house}/photo/${photoSelected}/url/${value}``